tokio/tests/net_unix_pipe.rs

```diff
@@ -298,9 +298,9 @@ async fn from_file_sets_nonblock() -> io::Result<()> {
     assert!(is_nonblocking(&reader)?);
 
     // Check if Sender sets the pipe in non-blocking mode.
-    let wronly = std::fs::OpenOptions::new().write(true).open(&fifo)?;
-    assert!(!is_nonblocking(&wronly)?);
-    let writer = pipe::Sender::from_file(wronly)?;
+    let wrongly = std::fs::OpenOptions::new().write(true).open(&fifo)?;
```

Comment:
Actually the author here meant `write only`, so maybe `wonly` or `w_only` or `write_only` ?!

Reply:
Thanks